soccer/gameplay/tactics/defense.py

```diff
@@ -14,7 +14,7 @@
 
 class Defense(composite_behavior.CompositeBehavior):
 
-    DEFENSE_ROBOT_CHANGE_COST = 0.29
+    DEFENSE_ROBOT_CHANGE_COST = 0.05
```

Comment:
@jgkamat Should we raise this? I remember a small issue from comp but i dont remember exactly

Reply:
I think it would help to have it a bit higher, the issue was flickering between the roles on the defense.  This is one of the things you have to test in the simulator to fully test.